spec/lib/date_time_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'date'
+require 'date_time'
+
+describe DateTime do
+    it 'get correct beginning of rice fiscal year' do
```

Comment:
I think 2 spaces for indent is the Accounts way

Reply:
Done